sql/sql_optimizer.cc

```diff
@@ -765,7 +765,8 @@ JOIN::optimize()
   if (!(select_options & SELECT_DESCRIBE) &&
       !select_lex->materialized_table_count)
   {
-    init_ftfuncs(thd, select_lex, order);
+    if (init_ftfuncs(thd, select_lex, order))
```

Comment:
All the other callers of init_ftfuncs do not check its return code - are they affected by this bug?